reporter/urlparams.js

```diff
@@ -85,7 +86,7 @@ function getUrlParams() {
 			// Allow just a key to turn on a flag, e.g., test.html?noglobals
 			value = param.length === 1 ||
 				decodeQueryParam( param.slice( 1 ).join( "=" ) ) ;
-			if ( urlParams[ name ] ) {
+			if ( hasOwn.call( urlParams, name ) ) {
```

Comment:
I think another possible approach would be to simply initialize the object like:

```js
var urlParams = Object.create(null);
```

Then we don't have to worry about any property collisions as there are none to inherit. Thoughts?